packages/esm-patient-registration-app/src/patient-registration/patient-registration.resource.ts

```diff
@@ -47,14 +47,33 @@ export function saveEncounter(encounter: Encounter) {
 export function generateIdentifier(source: string) {
   const abortController = new AbortController();
 
-  return openmrsFetch(`${restBaseUrl}/idgen/identifiersource/${source}/identifier`, {
+  return openmrsFetch(`${restBaseUrl}/idgen/identifiersource`, {
     headers: {
       'Content-Type': 'application/json',
     },
-    method: 'POST',
-    body: {},
+    method: 'GET',
```

Comment:
I think this is patching the wrong method altogether. This one is for generating identifiers and isn't the requests causing the issue.

Reply:
@ibacher thanks for the correction.  Its supposed to be  `generateIdentifier` method.